Description: Add a simple testcase
 Create a simple testcase that imports the module and prints something.
 .
 python-xtermcolor (1.2.1-1) unstable; urgency=low
 .
   * New upstream release
   * Updated patches to work with different path
   * Dropped vt100 patch, now part of upstream
Author: Salvo 'LtWorf' Tomaselli <tiposchi@tiscali.it>

--- /dev/null
+++ python-xtermcolor-1.2.1/test.py
@@ -0,0 +1,4 @@
+from xtermcolor import colorize
+
+print (colorize('Red on white',0xff0000,bg=0xffffff))
+
